chargebee/src/main/java/com/chargebee/android/billingservice/CBCallback.kt

```diff
@@ -17,5 +17,9 @@ interface CBCallback {
         fun onSuccess(result: ReceiptDetail, status: Boolean)
         fun onError(error: CBException)
     }
+}
 
+interface RestorePurchaseCallback {
+    fun onSuccess(result: List<CBRestoreSubscription>)
+    fun onError(error: CBException)
```

Comment:
Can this be inside `CBCallBack` to be consistent?

Reply:
Yes, done!